### DISCUSSION TITLE
DeleteRange operation: Delete all keys with prefix

### DISCUSSION THREAD

**Discussion by MaratKarimov** [oxia-db/oxia#586] [Q&A] [answered]
How can I delete all keys with a prefix using the DeleteRange operation?

**Comment by merlimat** [comment, marked as answer]
@MaratKarimov You can specify the highest key to delete. 

eg: If you have prefix separated by `/` you could do

```go
client.DeleteRange("/my-prefix/", "/my-prefix//")
```

If you have some sort of id: 

```go
client.DeleteRange("my-prefix-00000000", "my-prefix-ffffffff")
```
